src/components/DatePicker/DatePicker.js

```diff
@@ -69,11 +69,13 @@ export function DatePicker(props) {
             onChange={onMonthChange}
             className="ds-w-165px ds-py-5px ds-flex ds-px-14px ds-date-text ds-border-1.5 ds-border-multi-neutrals-grey85a ds-rounded"
           >
-            {monthValues.map((mv) => (
-              <option value={mv}>{language.datePicker.months[mv]}</option>
+            {monthValues.map((mv, index) => (
+              <option value={mv} key={index}>
```

Comment:
You shouldn't use **just** `index` as a key differentiator because react uses this to render/re-render; on a single page you may have conflicting ones that may create memory leaks. I suggest namespacing all instances on this to something more meaningful. In this case, you could have something like `datePicker-month-option-${index}`. Even better if you use a meaning `id` instead of `index`.

Reply:
change made according to request @zeddotes